en/docs/develop/enable-verification-for-updated-mobile-number.md

```diff
@@ -188,6 +207,35 @@ curl -X POST -H "Authorization: Basic Ym9iMTIzOnBhc3MxMjM=" -H "Content-Type: ap
 "HTTP/1.1 201 Created"
 ```
 
+Additionally can send the following curl command to resend a new SMS OTP code by a privileged user.
```

Comment:
```suggestion
Additionally, you can use the following curl command to resend a new SMS OTP code by a privileged user.
```

Reply:
The note about the WSO2 update should be added here

Reply:
Addressed with d9428901276377ae3f901fdab573f5006f7846ff